src/map/lua/lua_zone.cpp

```diff
@@ -453,7 +453,7 @@ void CLuaZone::Register()
     SOL_REGISTER("getID", CLuaZone::getID);
     SOL_REGISTER("getName", CLuaZone::getName);
     SOL_REGISTER("getRegionID", CLuaZone::getRegionID);
-    SOL_REGISTER("getType", CLuaZone::getType);
+    SOL_REGISTER("getType", CLuaZone::getTypeMask);
```

Comment:
This will want to be updated in all the Lua too:  SOL_REGISTER("getType", CLuaZone::getTypeMask);

Reply:
fidn and replace is failing me, brb with multiple corrections after this rebuild